### PR TITLE
Explain the publisherDocsUrl parameter in the manifest file

### PR DESCRIPTION
**Pull request by ashishguptaiitb**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/75c918d8fbde1b8280ee2923ccf79520965158ea/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 92 | 28 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/9a583f55-7154-40e0-b383-04384d584df7) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema.md [hunk 1]
```diff
@@ -808,7 +808,7 @@ When `defaultBlockUntilAdminAction` property is set to **true**, the app is hidd
 
 **Maximum size** - 128 characters
 
-The `publisherDocsUrl` is an HTTPS URL to an information page for admins to get guidelines before allowing an app, which is blocked by default. It can also be used to provide any instructions or information about the app, which can be useful for the tenant admin.
+The value of the `publisherDocsUrl` parameter is a secure, HTTPS URL to the app documentation and information page that app developers choose to provide. Specifically, tenant admins get documentation about the app via this URL in the Teams admin center. The documentation may include the instructions for admins before they allow the app to facilitate app adoption and app rollout. The URL destination can also include instructions or information about the app that is useful for the tenant admin, users, and other business stakeholders.
```
Comment (v-preethah) [inline msteams-platform/resources/schema/manifest-schema.md:811]: 1. Please remove comma after 'The value of the `publisherDocsUrl` parameter is a secure,'

2. Please change the sentence to: 
    The tenant admins get documentation about the app using this URL in the Teams admin center.

**Review by v-preethah — CHANGES_REQUESTED** [review]
Hi Ashish,

We have provided review comments. Request you to please make the necessary changes.

Thanks

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/690edfc7b8624700f916dfbc874014e79dceac94/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 92 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4554901-70e3-4b4a-927c-244778fdad47) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/413f41d978a225573f211f1cf935f617a8888221/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 92 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/49e861e9-d77b-4866-be76-492fbb58281c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema.md (hunk 1 above)
Comment (ashishguptaiitb) [reply msteams-platform/resources/schema/manifest-schema.md]: Thanks @v-preethah for the suggestions. Please verify the updates.

Comment (ashishguptaiitb) [reply msteams-platform/resources/schema/manifest-schema.md]: Hi @v-preethah  and @surbhigupta12 , please verify the changes and publish this PR if all's good.

**Review by srinidyMSFT — APPROVED** [review]

**Comment by ashishguptaiitb** [comment]
Hi @v-preethah , @Wajeed-msft , @v-habibaumme . Kindly approve so that this update can get published.
The feature PM has approved.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/106cc609daa97908f92d2db09afc12ab39f82386/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 92 | 27 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1261041b-7371-4234-9975-0b9b5f130e00) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by surbhigupta12 — APPROVED** [review]

**Review by v-preethah — APPROVED** [review]
Approved.

**Pull request merged**